native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -137,8 +139,27 @@ DecoderState::next(const double *probs,
               float score = 0.0;
               std::vector<std::string> ngram;
               ngram = ext_scorer_->make_ngram(prefix_to_score);
+
+              // when hot_boost is 1.0, there is no boost at all
+              float hot_boost = 1.0;
+              if (!hot_words_.empty()) {
+		std::map<std::string, float>::iterator iter;
+                // increase prob of prefix for every word
+                // that matches a word in the hot-words list
+                for (std::string word : ngram) {
+		  iter = hot_words_.find(word);
+                  if ( iter != hot_words_.end() ) {
+                    // increase the log_cond_prob(prefix|LM)
+                    // since the log_cond_prob is negative, we multiply by
+                    // a float <1.0 to increase.
+		    float boost = iter->second;
+                    hot_boost *= boost 
```

Comment:
so your PR is failing because you miss `;` here. Maybe rewrite:
```
hot_boost *= iter->second;
```

Since there is no reason for `float boost` declaration?

Reply:
agreed. fixed.